app/views/users/_invite.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if current_user.friend_requests.include?(user) %>
+  <%= link_to "Accept friendship", confirm_friend_path(user), class: 'profile-link' %>
+<% elsif !current_user.pending_friends.include?(user) && current_user!=user && !current_user.friend?(user) %>
+  <%= link_to "Invite to friendship", invite_friend_path(user), class: 'profile-link' %>
+<% end %>
```

Comment:
- [ ] View-related logic should be encapsulated in View helpers (app/helpers/*_helper.rb)

Reply:
Done.